app/Repositories/CacheStorage.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Gladiator\Repositories;
+
+use Illuminate\Support\Facades\Cache;
+
+trait CacheStorage
```

Comment:
@DFurnes brought up a similar point when I did the user cache stuff. While not necessarily needed at the moment, since we're doing our own implementations of some of the others that `Cache` methods that involve a bit of custom business logic for our app, I wanted to add these in here as well so that they're all in one place. Then we only really need to consider importing the `Cache` facade in one location and always know where to go to troubleshoot stuff if something's not working, instead of find their use scattered around in the separate `Cache*Repository` classes. More of a convenience thing for now, and if it turns out we do need to expand them w/ extra business logic, they're at the ready :)